tests/integration/nm/profile_test.py

```diff
@@ -654,3 +654,19 @@ def test_delete_mac_based_profile_using_profile_name(mac_based_profile_eth1):
         )[0]
         != 0
     )
+
+# https://issues.redhat.com/browse/RHEL-59239
+@pytest.mark.tier1
+def test_change_profile_name(eth1_up):
+    eth1_mac = get_mac_address("eth1")
+    desired_state = {
+        Interface.KEY: [
+            {
+                Interface.NAME: "eth1",
+                Interface.PROFILE_NAME: "eth1-new",
+                Interface.TYPE: InterfaceType.ETHERNET,
+                Interface.STATE: InterfaceState.UP,
+            }
+        ]
+    }
+    libnmstate.apply(desired_state)
```

Comment:
Can we add the assertion to assert that the profile name is `eth1-new` ?

Reply:
Add assertion. Thanks.